doc/api.yml

```diff
@@ -81,6 +81,8 @@ paths:
   #############################################
   ############## User Subscription API #############
   #############################################
+  /users/stripe/subscription:
```

Comment:
users/subscriptionRegistration

Remove stripe from the API, as it is only a third-party service that must no be known by consumer.